docs/src/app/components/pages/components/GridList/Page.jsx

```diff
@@ -12,7 +12,7 @@ import gridListExampleComplexCode from '!raw!./ExampleComplex';
 import GridListExampleComplex from './ExampleComplex';
 
 const descriptions = {
-  simple: 'A simple example of a scrollable `GridList`.',
+  simple: 'A simple example of a scrollable `GridList` containing a `Subheader`.',
```

Comment:
Yes, we discussed it for some other places too. I'll provide links in my next commit :sweat_smile: :sweat_smile: